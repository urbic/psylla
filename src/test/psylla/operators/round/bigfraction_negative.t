-26328848881252255650756946135039519281 13 div round ?
