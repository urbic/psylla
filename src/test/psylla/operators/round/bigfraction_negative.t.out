-2025296067788635050058226625772270713
